Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -448,7 +448,16 @@ public void Init(IServiceCollection serviceCollection)
 
             ConfigurationManager.AddParts(GetExports<IConfigurationFactory>());
 
-            NetManager = new NetworkManager(ConfigurationManager, _startupConfig, LoggerFactory.CreateLogger<NetworkManager>());
+            // this is needed as Github Codespaces will not be able to find the process if we bind on interfaces directly.
+
+            if (bool.TryParse(Environment.GetEnvironmentVariable("JELLYFIN_NETWORKMANAGER_BINDALL"), out var bindAll) && bindAll)
+            {
+                NetManager = new BindAllNetworkManager(ConfigurationManager, _startupConfig, LoggerFactory.CreateLogger<NetworkManager>());
+            }
+            else
+            {
+                NetManager = new NetworkManager(ConfigurationManager, _startupConfig, LoggerFactory.CreateLogger<NetworkManager>());
+            }
```

Comment:
I don't understand why this is necessary. Why not [add the explicit port definition](https://github.com/jellyfin/jellyfin-vue/blob/master/.devcontainer/devcontainer.json#L14)?

Reply:
i tried that but as i described in the readme docs, GH codespaces will not bind the proxy correctly to the ports. I think at the time JF starts the virtual interface might not be available yet and it binds its later and then JF does not react to it.

Reply:
It seems like the issue was already addressed by chance 2 months ago. i could remove the bind-all flag for what its worth but i sometimes see it pop up as an feature request so i would like to keep it anyway. Thoughts?